wiki/Tournaments/SS_Tournament/2018/en.md

```diff
@@ -0,0 +1,294 @@
+
+
+## Mappools
+
+### Qualifiers
+
+![Qualifiers Mappool](Qaulifiers.jpg)
+
+- NoMod
+  - [Ceui - COLORFUL DAYS!!](https://osu.ppy.sh/beatmaps/203315) [Insane]
+  - [Mastermind(xi+nora2r) - Dreadnought](https://osu.ppy.sh/beatmaps/813331) [Another]
+  - [Function Phantom - Euclid](https://osu.ppy.sh/beatmaps/971086) [toybot's Insane]	
+- DoubleTime
+  - [Duca - Clover](https://osu.ppy.sh/beatmaps/157856) [Collab]	
+  - [Icon For Hire - Make A Move](https://osu.ppy.sh/beatmaps/1077369) [Hard]		
+- Hidden  
+  - [Dreamcatcher - Sleep-walking](https://osu.ppy.sh/beatmaps/1370703) [Insane]	
+  - [yuiko - petit bonheur](https://osu.ppy.sh/beatmaps/1327955) [Insane]	
+- Hardrock
+  - [Otokaze - Karen (Short Ver.)](https://osu.ppy.sh/beatmaps/154357) [Hanabi]		
+  - [ALiCE'S EMOTiON - Dark Flight Dreamer](https://osu.ppy.sh/beatmaps/763294) [Twaoi's Insane]			
+### Group Stage
+
+![Group Stage Mappool](Group.jpg)
+
+- NoMod
+  - [GARNiDELiA - ambiguous](https://osu.ppy.sh/beatmaps/680886) [Insane]
+  - [Mind Vortex - Arc](https://osu.ppy.sh/beatmaps/239037) [NSane]
+  - [sakuzyo - VALLISTA](https://osu.ppy.sh/beatmaps/127313) [Another]
+  - [B-ko(Cv:Touyama Nao) - Nisemono Chuuihou](https://osu.ppy.sh/beatmaps/1253351) [Insane]
+  - [Mediks - Doomsday](https://osu.ppy.sh/b/1290316) [Insane]
+  - [Dark PHOENiX - Taketori Hishou](https://osu.ppy.sh/beatmaps/86324) [Extra]
+- Double Time
+  - [Two Steps From Hell - Star Sky](https://osu.ppy.sh/beatmaps/1270965) [Unstoppable]
+  - [Petit Rabbit's - No Poi!(nenpulse bootleg remix)](https://osu.ppy.sh/beatmaps/937188) [YakiBoss's Hard]
+  - [Chata - Koi wo Shiyouyo](https://osu.ppy.sh/beatmaps/81200) [Insane]
+- Hidden
+  - [Kano - Yuudachi no Ribbon](https://osu.ppy.sh/beatmaps/1078420) [Dailycare's Insane]
+  - [Renard - Terminal](https://osu.ppy.sh/beatmaps/270363) [Ends' Insane]
+  - [Ne;on & Haiji - Haruka Tooki Gensoukyo - Dream of fantasy](https://osu.ppy.sh/beatmaps/106475) [lkp's Lunatic]
+- Hardrock
+  - [Rchetype - V Is For](https://osu.ppy.sh/beatmaps/290919) [Insane]
+  - [MOMOIRO CLOVER Z - SARABA ITOSHIKI KANASHIMI TACHIYO](https://osu.ppy.sh/beatmaps/507098) [Master]
+  - [Sota Fujimori - Move That Body -Extended Mix-](https://osu.ppy.sh/beatmaps/547229) [RLC's Insane]
+- Free Mod
+  - [Kozato Snow - Izayoi Sakura](https://osu.ppy.sh/beatmaps/86352) [Insane]
+  - [iojjj - Deus Ex Machina](https://osu.ppy.sh/beatmaps/688907) [EXTRA]
+  - [Mutsuhiko Izumi - Red Goose](https://osu.ppy.sh/beatmaps/144028) [Another]
+- Tiebreaker
+  - [Demetori - The Grimoire of Alice ~ Alice in Wonderland](https://osu.ppy.sh/beatmaps/628369) [Lolis]
+
+### Round Of 16
+
+![Round Of 16 Mappool](Round.jpg)
+
+ - NoMod
+  - [Eiko Shimamiya - Higurashi no Naku Koro ni](https://osu.ppy.sh/beatmaps/738088) [Extra]
```

Comment:
there is an indentation error for these sub-bullets, add another space before the `-` (repeat for the others in this NoMod list)